src/datepicker/datepicker.js

```diff
@@ -228,7 +226,13 @@ angular.module('ui.bootstrap.datepicker', ['ui.bootstrap.dateparser', 'ui.bootst
       past: time < 0,
       current: time === 0,
       future: time > 0,
-      customClass: this.customClass(date) || null
+      customClass: this.customClass(date) || null,
+      long: date.toLocaleDateString("en-US",{
+        day:"numeric",
+        weekday:"long",
+        year: "numeric",
+        month: "long"
+      })
```

Comment:
this creates a human readable date string for VO/SR. Right now the local is capped to US but so is our site. When we got after i18n this can be updated fairly easily based on code i've seen in other date picker libraries.